run-tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Try to install composer dev dependencies
+composer install --no-interaction --no-scripts --no-plugins
+
+# If that failed, exit.
+rc=$?; if [[ $rc != 0 ]]; then exit $rc; fi
+
+# Run the feature tests
+vendor/bin/behat --stop-on-failure
```

Comment:
we need runny for commands that bash will ignore errors on and proceed to next command, but the last command in the file does not need runny so we can see its full output and its exit code will stop a build if it fails anyway